controllers/util/k8sutil.go

```diff
@@ -44,6 +45,8 @@ const (
 	OwnerUIDIndexName = "ownerUID"
 )
 
+var CRDList []string
```

Comment:
Global singleton non-cost, non-atomic state inside a utils file is a red flag. Why do we need this?